lib/pbench/server/api/resources/users_api.py

```diff
@@ -163,9 +144,10 @@ def __init__(self, config, logger, auth):
                 "pbench-server", "token_expiration_duration"
             )
         except NoOptionError:
-            self.token_expire_duration = None
+            # Defaults to 60 minutes
+            self.token_expire_duration = 60
 
-    @Auth.token_auth.login_required(optional=True, f=Auth().verify_auth())
+    @Auth.token_auth.login_required(optional=True)
```

Comment:
Sorry, Nikhil, but I'm still not seeing it:  how does this cause `Auth.verify_auth()` to be invoked?  (If it doesn't, then what does?)

Reply:
`login_required()` decorator invokes the method defined with another decorator `verify_token`. In our case `verify_auth(auth_token)` is the one that is defined with `verify_token` decorator inside Auth module. The verify_token callback receives the authentication credentials provided by the client on the Authorization header (In our case bearer token). Ref: [flask_httpauth](https://flask-httpauth.readthedocs.io/en/latest/)

Reply:
Once again, thank you for the education, Nikhil.  So, it's baked in (to Flask, this time...) that a function decorated with `login_required` will call another function which was decorated with `verify_token` (at least for the token-based authorization case).  Python magic, this time resulting in code obfuscation.  :-(